.circleci/scripts/yarn-audit

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -u
+set -o pipefail
+
+yarn audit --level moderate
+audit_status="$?"
+
+# Use a bitmask to ignore INFO and LOW severity audit results
+# See here: https://yarnpkg.com/lang/en/docs/cli/audit/
+audit_status="$(( audit_status & 11100 ))"
+
+if [[ "$audit_status" != 0 ]]
+then
+    count="$(yarn audit --json | tail -1 | jq '.data.vulnerabilities.moderate + .data.vulnerabilities.high + .data.vulnerabilities.critical')"
```

Comment:
This still isn't counting the correct thing - it needs to discard dev dependencies. I'll fix this.

Reply:
It looks like `yarn` is incorrectly reporting our development dependencies as production dependencies in these security audits 😞. Until this is fixed, we may have to hold off in migrating to yarn.

Reply:
Nevermind, found a workaround! The flag `--groups dependencies` does effectively narrow the results to just the production dependencies.

There is still a bug in the advisory output... but that's not something we need to rely upon thankfully.